Sprint-3/quote-generator/quotes.js

```diff
@@ -491,3 +491,19 @@ const quotes = [
 ];
 
 // call pickFromArray with the quotes array to check you get a random quote
+
+function generateNewQuote() {
+  const newQuote = pickFromArray(quotes);
+  const quoteText = newQuote.quote;
+  const quoteAuthor = newQuote.author;
+
+  document.getElementById("quote").innerText = `"${quoteText}"`;
```

Comment:
Good work. Typically you would set this to a variable so that it is reusable. Y0ou could also do something like this: 
  quote.textContent = output.quote;
  author.textContent = "-" + output.author;